### PR TITLE
fix: call the correct migration function for Admin Queries

### PR DESCRIPTION
**Pull request by jhockett**
#### Checklist

- [ ] PR description included
- [x] `yarn test` passes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.ts [hunk 1]
```diff
@@ -597,6 +597,28 @@ async function askLambdaArn(context: $TSContext, currentPath?: ApigwPath) {
...
+    if (!adminQueriesResource?.dependsOn) {
+      throw new Error('Failed to migrate Admin Queries API. Could not find expected information in amplify-meta.json.');
+    }
+
+    const { dependsOn } = adminQueriesResource;
+    const functionName = dependsOn.filter(dependency => dependency.category === AmplifyCategories.FUNCTION)?.[0]?.resourceName;
+
+    const adminQueriesProps = {
+      apiName: resourceName,
+      authResourceName: getAuthResourceName(),
```
Comment (akshbhu) [inline packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.ts:614]: await getAuthResourceName()

Comment (jhockett) [reply packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.ts:614]: this is a synchronous function defined locally in this file

**Review by johnpc — APPROVED** [review]

**Review by akshbhu — APPROVED** [review]
lgtm

File: packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.ts [hunk 2]
```diff
@@ -597,6 +597,28 @@ async function askLambdaArn(context: $TSContext, currentPath?: ApigwPath) {
 
 export async function migrate(context: $TSContext, projectPath: string, resourceName: string) {
   const apigwInputState = new ApigwInputState(context, resourceName);
+
+  if (resourceName === ADMIN_QUERIES_NAME) {
+    const meta = stateManager.getMeta();
+    const adminQueriesResource = meta?.[AmplifyCategories.API]?.[ADMIN_QUERIES_NAME];
```
Comment (attilah) [inline packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.ts:603]: use lodash to get dependsOn straight ahead?

**Pull request merged**